src/util/login.js

```diff
@@ -0,0 +1,51 @@
+import * as firebase from 'firebase';
```

Comment:
Add a comment to the top of this file giving a description of what kinda of functions it contains.